ci/continuous-builder.sh

```diff
@@ -84,4 +130,10 @@ fi
 # Get updates to ali-bot
 TIMEOUT=$(get_config_value timeout "${TIMEOUT:-600}") reset_git_repository ali-bot
 
-exec continuous-builder.sh
+# Run CI builder under screen if possible and we're not already there. This is
+# for the macOS builders.
+if [ -n "$STY" ] || ! type screen > /dev/null; then
+  exec "$0" --skip-setup "$@"
+else
+  exec screen -dmS "ci_$WORKER_INDEX" "$0" --skip-setup "$@"
```

Comment:
I never liked the idea of running under screen. Can you check how we can make sure we run under systemd?

Reply:
@MichaelLettrich could you send me the old launchd scripts? I'll see what I can do with those.

@ktf it might be better to keep screen for now so the builds work again, then I can migrate to launchd some time this week when there's less time pressure.